api/src/modules/impact/dto/get-impact-table.dto.ts

```diff
@@ -57,4 +58,10 @@ export class GetImpactTableDto {
   @IsUUID(4, { each: true })
   @Type(() => String)
   supplierIds?: string[];
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsEnum(LOCATION_TYPES, { each: true })
+  @Type(() => String)
+  locationTypes?: LOCATION_TYPES[];
```

Comment:
Shouldn't this be singular? Also can you please add relevant documentation to swagger to let the consumer know which are the available filters?